frontend/css/fresh-releases.less

```diff
@@ -314,6 +314,9 @@
   .sidebar {
     flex: 0 0 250px;
   }
+  .sidebar-fresh-releases {
+    max-height: 85vh;
+  }
```

Comment:
```suggestion
  .sidebar-fresh-releases {
    max-height: 85vh;
    
    @media screen and (max-width: @screen-md) {
      max-height: calc(100vh - @brainzplayer-height);
    }    
  }
```

I believe you should also add media queries to improve the view on smaller screen devices.

<img width="383" alt="image" src="https://github.com/user-attachments/assets/ba5e349e-b570-46fa-a0bb-e45a71ccea09" />